### DISCUSSION TITLE
Add SOCKS5 proxy support

### DISCUSSION THREAD

**Discussion by TwinkleToes777** [status-im/status-legacy#16627] [General]
# Feature

### User Story

As a user I want to be able to configure a SOCKS5 proxy to connect to Status, this helps to bypass blockage.

### Description
*Summary*: ...
Add a SOCKS5 proxy support, a configurable IP, port and optionally username and password in the app.
### Acceptance Criteria
All connections must be routed through SOCKS5 proxy when configured.

### Notes
None

**Comment by oskarth** [comment]
See related https://github.com/status-im/nim-libp2p/issues/358

**Comment by cammellos** [comment]
Moving to a discussion, since that will require work mostly on different projects
